.travis.yml

```diff
@@ -128,7 +128,7 @@ matrix:
         - redis-server
         - postgresql
     - python: 2.7
-      env: TEST_SUITE=snuba SNUBA=http://localhost:8000
+      env: TEST_SUITE=snuba SENTRY_TAGSTORE=sentry.tagstore.snuba.SnubaTagStorage SNUBA=http://localhost:8000
```

Comment:
Ya, I can go ahead and do that to future proof. Right now though `tagstore` is being implicitly and the only test against search is manually instantiating `SnubaSearchBackend` (then implicitly using `tagstore`).

That said, it can't hurt and we'll want it in the future anyway.